orchagent/portsorch.cpp

```diff
@@ -3671,6 +3646,36 @@ bool PortsOrch::addBridgePort(Port &port)
 
     if (port.m_bridge_port_id != SAI_NULL_OBJECT_ID)
     {
+        /* If the port is being added to the first VLAN,
```

Comment:
I didn't quite understand this section. Is this for the previous bridge port? In my understanding, this API is for the first bridge port addition, correct?